foursight_core/deploy.py

```diff
@@ -97,6 +99,8 @@ def build_config(cls, stage, trial_creds=None, trial_global_env_bucket=False, gl
                 curr_stage_environ['ENV_NAME'] = env_name
             if es_host:
                 curr_stage_environ['ES_HOST'] = es_host
+            if s3_key_id:
+                curr_stage_environ['S3_ENCRYPT_KEY_ID'] = s3_key_id
```

Comment:
Ditto, but there may be other references (as in `s3_connection.py`). I'll stop marking them.